sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -108,6 +108,27 @@ export class Sender {
   /**
    * Schedules given message to appear on Service Bus Queue/Subscription at a later time.
    *
+   * Recently, a bug has been surfaced with the `scheduleMessage()` method.
+   *
+   * More reference and details on the bug - https://github.com/Azure/azure-sdk-for-js/issues/6816#issuecomment-574461068
+   *
+   * You are affected with the bug in case you are depending on version < 2.0.0 of `@azure/service-bus`.
+   *
+   * Version 2.0.0 is not released yet and the fix for this bug will be shipped along with the release.
```

Comment:
If this is going to be released with v 2.0, then why are we bumping minor version? Is this to go in on track 2 branch instead?

Reply:
In the minor version, only the docs are updated with the workaround, we are not fixing the bug.
When we release 2.0.0, we will fix the bug and remove this workaround.

Reply:
Oh oki, makes sense!